dataprofiler/tests/profilers/test_categorical_column_profile.py

```diff
@@ -283,12 +283,10 @@ def test_categorical_merge(self):
                                                   '4', 'b', '3', 'dfd', 'ee',
                                                   'ff', 'nan', 'None', '1',
                                                   'gg', 'null', 'NaN'])
-        expected_dict = dict({'aa': 5, '2': 4, 'abcd': 4, 'b': 3, np.nan: 2})
-        self.assertEqual(len(report_count), len(expected_dict))
-        self.assertEqual(report_count['aa'], expected_dict['aa'])
-        self.assertEqual(report_count['2'], expected_dict['2'])
-        self.assertEqual(report_count['abcd'], expected_dict['abcd'])
-        self.assertEqual(report_count['b'], expected_dict['b'])
+        expected_dict = {'aa': 5, '2': 4, 'abcd': 4, 'b': 3, np.nan: 2,
+                         'dfd': 2, '3': 2, '4': 2, 'ee': 2, 'null': 1, 'ff': 1,
+                         'NaN': 1, '1': 1, 'nan': 1, 'gg': 1, 'None': 1}
+        self.assertCountEqual(report_count, expected_dict)
```

Comment:
will approve, but for future, the tests should be unless the assert specifically says otherwise in the docs `self.assert...(expected, actual)`